Test failure on 32bit architectures

### Describe the bug

Attempting to run the tests on any 32bit architecture results in the following error:

```
# github.com/rabbitmq/amqp091-go_test [github.com/rabbitmq/amqp091-go.test]
src/github.com/rabbitmq/amqp091-go/examples_test.go:535:39: cannot use 5_000_000_000 (untyped int constant 5000000000) as int value in map literal (overflows)
FAIL	github.com/rabbitmq/amqp091-go [build failed]
FAIL
```

Explicitly defining that constant as an `int64` fixes the issue:

```
diff --git a/examples_test.go b/examples_test.go
index d83a824..98872ed 100644
--- a/examples_test.go
+++ b/examples_test.go
@@ -532,7 +532,7 @@ func ExampleChannel_QueueDeclare_stream() {
 		false,             // noWait
 		amqp.Table{ // queue args
 			amqp.QueueTypeArg:                 amqp.QueueTypeStream,
-			amqp.StreamMaxLenBytesArg:         5_000_000_000, // 5 Gb
+			amqp.StreamMaxLenBytesArg:         int64(5_000_000_000), // 5 Gb
 			amqp.StreamMaxSegmentSizeBytesArg: 500_000_000,   // 500 Mb
 			amqp.StreamMaxAgeArg:              "3D",          // 3 days
 		},
```

(Seen while updating the Debian packaging for this library.)

### Reproduction steps

Run the tests on an i386, armel, or armhf system.

### Expected behavior

Everything works on both 32 and 64 bit systems. :)

### Additional context

_No response_

--- a/examples_test.go
+++ b/examples_test.go
@@ -532,9 +532,9 @@ func ExampleChannel_QueueDeclare_stream() {
 		false,             // noWait
 		amqp.Table{ // queue args
 			amqp.QueueTypeArg:                 amqp.QueueTypeStream,
-			amqp.StreamMaxLenBytesArg:         5_000_000_000, // 5 Gb
-			amqp.StreamMaxSegmentSizeBytesArg: 500_000_000,   // 500 Mb
-			amqp.StreamMaxAgeArg:              "3D",          // 3 days
+			amqp.StreamMaxLenBytesArg:         int64(5_000_000_000), // 5 Gb
+			amqp.StreamMaxSegmentSizeBytesArg: 500_000_000,          // 500 Mb
+			amqp.StreamMaxAgeArg:              "3D",                 // 3 days
 		},
 	)
 	log.Printf("Declared queue: %s", q.Name)

